support-frontend/assets/pages/subscriptions-landing/components/subscriptionsLandingContent.jsx

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+import React from 'react';
+
+import { connect } from 'react-redux';
+
+// components
+import SubscriptionsProduct from './subscriptionsProduct';
+import FeatureHeader from './featureHeader';
+
+// images
+import FeaturePackshot from 'components/packshots/feature-packshot';
+import GuardianWeeklyPackShot from 'components/packshots/guardian-weekly-packshot';
+import PaperPackshot from 'components/packshots/paper-packshot';
+import PremiumAppPackshot from 'components/packshots/premium-app-packshot';
+import PaperAndDigitalPackshot from 'components/packshots/paper-and-digital-packshot';
+
+// helpers
+import { displayPrice, sendTrackingEventsOnClick } from 'helpers/subscriptions';
+import { getCampaign } from 'helpers/tracking/acquisitions';
+import { getSubsLinks } from 'helpers/externalLinks';
+import { androidAppUrl, getIosAppUrl } from 'helpers/externalLinks';
+import trackAppStoreLink from 'components/subscriptionBundles/appCtaTracking';
+
+// types
+import { type CommonState } from 'helpers/page/commonReducer';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { type Participations } from 'helpers/abTests/abtest';
+import { type OptimizeExperiments } from 'helpers/optimize/optimize';
+import { type ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
+
+type PropTypes = {
+  countryGroupId: CountryGroupId,
+  referrerAcquisitionData: ReferrerAcquisitionData,
+  abParticipations: Participations,
+  optimizeExperiments: OptimizeExperiments,
+}
+
+const SubscriptionsLandingContent = ({
+  countryGroupId, referrerAcquisitionData, abParticipations, optimizeExperiments,
+}: PropTypes) => {
+
+  const subsLinks = getSubsLinks(
+    countryGroupId,
+    referrerAcquisitionData.campaignCode,
+    getCampaign(referrerAcquisitionData),
+    referrerAcquisitionData,
+    abParticipations,
+    optimizeExperiments,
+  );
+  const abTest = null;
```

Comment:
Shouldn't this be assigned a value from somewhere? It looks like it's always null at the moment